zeebe/benchmarks/project/src/main/java/io/camunda/zeebe/config/StarterCfg.java

```diff
@@ -40,6 +40,16 @@ public class StarterCfg {
   private boolean startViaMessage;
   private String msgName;
 
+  private boolean monitorApiVisibility;
```

Comment:
❓ Would it maybe enought to just check for the Operateurl whether this is set?

Reply:
Then we get rid of the default value, no? I don't have a strong opinion here tbh, because I can't think of a case right now where you'd configure the Operate URL and disable this :shrug: